ts/graphs/buttons.ts

```diff
@@ -134,6 +135,18 @@ export function renderButtons(
         return { total, correct, percent };
     };
 
+    const totalPressedStr = (data: Datum): string => {
+        const groupTotal = totalCorrect(data.group).total;
+        const buttonTotal = data.count;
+
+        const buttonTotalCommaSeparated = localizedNumber(buttonTotal);
+        const percent = buttonTotal
+            ? ((buttonTotal / groupTotal) * 100).toFixed(2)
+            : "0";
```

Comment:
We probably need to localize the percent number as well, as otherwise the thousands separator doesn't match. You can test with `scripts/ts-run -l de`. There are other areas that are inconsistent and need updating as well, but that belongs in a separate PR (if you're interested please let us know; if not it may be something @RumovZ might like to tackle)

Reply:
I wouldn't mind going for a hunt for these. Seems like a good way to explore the codebase